lib/screens/welcome_screen.dart

```diff
@@ -29,6 +29,8 @@ class _WelcomeScreenState extends State<WelcomeScreen> {
   @override
   Widget build(BuildContext context) {
     MediaQueryData queryData;
+    Color yellow = const Color(0xffE8CC09);
+    Color purple = const Color(0xff5143A5);
```

Comment:
Add these two colors in `/utils/constants.dart` as
```dart
const Color kRegisterButtonColor = Color(0xffE8CC09);
const Color kLoginButtonColor = Color(0xff5143A5);
```
In the long run, this will make refactoring code easier. You can also apply this color in multiple places throughout the app.